packages/pds/src/index.ts

```diff
@@ -186,6 +186,18 @@ export class PDS {
     }
 
     app.use(health.createRouter(ctx))
+    app.get('/', function (req, res) {
+      res.type('text/plain')
+      res.send(
+        'This is an AT Protocol Personal Data Server (PDS): https://github.com/bluesky-social/atproto\n\nMost API routes are under /xrpc/',
+      )
+    })
+    app.get('/robots.txt', function (req, res) {
+      res.type('text/plain')
+      res.send(
+        '# Hello!\n\n# Crawling the public API is allowed\nUser-agent: *\nAllow: /',
```

Comment:
does the pds need to be indexed, esp. when auth is required for some endpoints?

Reply:
I'm not sure what you mean, but what this part of the PR is trying to make more explicit is that anything which is not auth'd can be crawled.

In the atproto/bluesky system, public repo content can be crawled publicly.

Reply:
well Twitter's API isn't crawlable, actually explicitly. so it makes more sense for the actual client to be crawlable because it has the posts themselves. when I search "hellthread" or something, I expect to see an instance on there and not the JSON representation of that. also see #1000

Reply:
As one example, some web clients are Javascript "single page apps", which then do HTTP API calls to this backend service directly from the browser, with no authentication. This is distinct from any service which does server-to-server HTTP API request.

In that case, if somebody wanted to use web.archive.org or similar to crawl, using a browser-style javascript-aware crawler, the crawler (bot) would be making HTTP requests to this service.